extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -136,7 +136,7 @@ class CornerstoneViewportService implements IViewportService {
   public destroy() {
     this._removeResizeObserver();
     this.viewportGridResizeObserver = null;
-    this.renderingEngine.destroy();
+    this.renderingEngine?.destroy?.();
```

Comment:
The renderingEngine was null, so the destroy dereference threw an exception.  This caused failures to destroy the rest of the data.